package.json

```diff
@@ -4,8 +4,10 @@
   "description": "A reliable way to get env",
   "main": "lib/index.js",
   "scripts": {
-    "test": "apm test",
-    "compile": "ucompiler go"
+    "test": "(apm test) && (flow check) && (eslint . )",
```

Comment:
Oh boy, I really really like this as one task 😛 
Besides when you are developing with Atom you do linting in it anyway, and when you are in console, you wanna test everything with just one command